app.R

```diff
@@ -364,9 +365,23 @@ server <- function(input, output, session) {
     make_dt(dt, table_view_opts = 'ftp')
   })
 
+  gpu_records_colnames <- reactive({
+    df <- gpu_records()
+    cols <- colnames(df) %>%
+      setdiff(c('timestamp', 'USER_NAME', 'Job', 'JOB_ID', 'QUEUE_NAME'))
+
+    pretty_cols <- str_replace_all(cols, '_', ' ') %>% str_to_title()
```

Comment:
I am getting this error
`Error in str_to_title: could not find function "str_to_title"`
We need to add `stringr::` before `str_replace_all` and `str_to_title` I guess.

We also need to include new `loadNamespace` statement in the begining to highlight dependency.